connector_salesforce/README.rst

```diff
@@ -0,0 +1,247 @@
+Salesforce Connector
```

Comment:
Please follow the OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
Please add tags/images at the beginning of the README + document title should be be underlined before.